source/fileInfo.go

```diff
@@ -177,11 +189,15 @@ func (fi *FileInfo) init() {
 		if (!fi.isLeafBundle && len(parts) == 1) || len(parts) > 1 {
 			section = parts[0]
 		}
-
 		fi.section = section
 
-		fi.uniqueID = helpers.MD5String(filepath.ToSlash(fi.relPath))
+		var contentBaseName = fi.translationBaseName
```

Comment:
Simplify this section to avoid writing to `contentBaseName` twice on leaves.

```go
if fi.isLeafBundle && len(parts) > 0 {
    fi.contentBaseName = parts[len(parts)-1]
} else {
    fi.contentBaseName = fi.translationBaseName
}
```

Reply:
My initial implementation actually used direct assignment, but then I spotted this local variable pattern in the lines just above (implementing `section`), and thought it might be a golang idiom...

I'll change it back to direct assignment which I also find easier to read.